test/contracts/core/fee-calculators/performanceFeeCalculator.spec.ts

```diff
@@ -566,6 +568,109 @@ contract('PerformanceFeeCalculator', accounts => {
         expect(feeState.profitFeePercentage).to.be.bignumber.equal(newFeePercentage);
       });
 
+      describe('when the profit fee is initially 0 and a profit is marked', async () => {
```

Comment:
Don't think we need the and a profit is marked phrase here since it tests that and the complementary case below